yarn.lock

```diff
@@ -2116,7 +2116,7 @@
   resolved "https://registry.yarnpkg.com/@mapbox/geojson-normalize/-/geojson-normalize-0.0.1.tgz#1da1e6b3a7add3ad29909b30f438f60581b7cd80"
   integrity sha1-HaHms6et060pkJsw9Dj2BYG3zYA=
 
-"@mapbox/geojson-rewind@^0.4.0", "@mapbox/geojson-rewind@^0.4.1":
+"@mapbox/geojson-rewind@^0.4.1":
```

Comment:
It looks like this transitive dependency is no longer needed, can you remove it?

Reply:
Weird - I would have thought that Yarn would remove unused dependencies from `yarn.lock` 🤔 But yeah, it looks like `@mapbox/geojson-rewind@^0.4.1` isn't in use

Reply:
Well, I guess `@mapbox/geojson-rewind@^0.4.1` isn't used it `master` either, so it's technically not part of this PR 🤷‍♂️ Must have been a bad merge previously maybe?

Reply:
My bad - it's because it's a direct dependency (I.e. not a sub-dependency), which is why it's not referenced anywhere else in `yarn.lock`

Reply:
🤦sorry about that